CMakeLists.txt

```diff
@@ -1724,8 +1724,8 @@ if(ENGINEPRIME)
     set(DJINTEROP_LIBRARY "lib/${CMAKE_STATIC_LIBRARY_PREFIX}djinterop${CMAKE_STATIC_LIBRARY_SUFFIX}")
     file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/download")
```

Comment:
Is this directory still used?

Reply:
> I get now obscure error messages, but no significant status message for a hint how to treat this situation:
> 

The detached checkout warnings appear always. Don't know how to suppress them.

Reply:
The DOWNLOAD_DIR is used in the same ways as before.